packages/addons/service/prometheus-node-exporter/source/system.d/service.prometheus-node-exporter.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Prometheus Node Exporter service
+After=network-online.target
+Requires=network-online.target
+
+[Service]
+ExecStart=/bin/sh -c "exec sh /storage/.kodi/addons/service.prometheus-node-exporter/bin/prometheus-node-exporter.start"
```

Comment:
```suggestion
ExecStart=/bin/sh /storage/.kodi/addons/service.prometheus-node-exporter/bin/prometheus-node-exporter.start
```
Or even 
```
ExecStart=/storage/.kodi/addons/service.prometheus-node-exporter/bin/prometheus-node-exporter.start
```
because we perform `chmod +x <addon dir>/bin/*` on installation.

Reply:
Fixed , thx !